sympy/stats/tests/test_finite_rv.py

```diff
@@ -395,10 +441,16 @@ def test_FiniteRV():
         *[Eq(F.symbol, i) for i in [1, 2, 3]])
 
     assert F.pspace.domain.set == FiniteSet(1, 2, 3)
-    raises(ValueError, lambda: FiniteRV('F', {1: S.Half, 2: S.Half, 3: S.Half}))
-    raises(ValueError, lambda: FiniteRV('F', {1: S.Half, 2: Rational(-1, 2), 3: S.One}))
+    raises(ValueError, lambda: FiniteRV('F', {1: S.Half, 2: S.Half, 3: S.Half}, check=True))
+    raises(ValueError, lambda: FiniteRV('F', {1: S.Half, 2: Rational(-1, 2), 3: S.One}, check=True))
     raises(ValueError, lambda: FiniteRV('F', {1: S.One, 2: Rational(3, 2), 3: S.Zero,\
-        4: Rational(-1, 2), 5: Rational(-3, 4), 6: Rational(-1, 4)}))
+        4: Rational(-1, 2), 5: Rational(-3, 4), 6: Rational(-1, 4)}, check=True))
+
+    # purposeful invalid pmf but it should not raise since check=False
+    # see test_drv_types.test_ContinuousRV for explanation
```

Comment:
Why `test_ContinuousRV` would be in `test_drv_types.py`

Reply:
I've only added `test_ideal_soliton` and `test_robust_soliton`. Other changes are already part of master. Maybe its a typo and it should be `test_crv_types.test_ContinuousRV` in the comment instead. Let me know and I'll commit the changes if required

Reply:
May be merge `master` into your branch, so that irrelevant changes aren't shown here. You can fix these typos in a different PR.

Reply:
OK, I'll do that. Should I do it now? I'm asking because Travis builds seem to taking a very long time. Maybe doing it beforing merging would be better. Or I'll merge `master` and you could cancel the build as it would be redundant. I'll wait for the current build to pass

Reply:
You can do it now. This will be merged at least after 24 hours.

Reply:
Done, the diff is clean now